.github/workflows/publish-alpha.yml

```diff
@@ -0,0 +1,32 @@
+name: "publish to npm"
```

Comment:
Maybe this to make it clear which npm command is being run:
```suggestion
name: "publish alpha release to npm"
```

Reply:
Nice catch!